src/providers/ad/ad_gpo.c

```diff
@@ -4777,14 +4783,14 @@ gpo_fork_child(struct tevent_req *req)
     if (ret == -1) {
         ret = errno;
         DEBUG(SSSDBG_CRIT_FAILURE,
-              "pipe failed [%d][%s].\n", errno, strerror(errno));
+              "pipe (from) failed [%d][%s].\n", errno, strerror(errno));
```

Comment:
In some places for strigns we are using '%s' and in other places [%s]. If there is some general rule when square brackets ([]) should be used instead of apostrophes ('')?

Reply:
I'm not aware of a general rule.
Usually I tend to keep what was already in place.